src/LoginForm/LoginForm.js

```diff
@@ -52,6 +58,9 @@ class LoginForm extends Component {
           >Other</button>
         </div>
         <button onClick={this.submitForm}>Explore</button>
+        <h4
+        className={this.state.formFilled ? 'input-error':'hidden'}
```

Comment:
I like the way you added formFilled to state rather than relying on a variable.  Great use of state and class component!